core/ui/card_ui/disks/steam_removable_media.gd

```diff
@@ -0,0 +1,66 @@
+extends ScrollContainer
+
+var steam_disks := load("res://core/systems/disks/steam_removable_media_manager.tres") as SteamRemovableMediaManager
+
+const drive_card_scene: PackedScene = preload("res://core/ui/components/drive_card.tscn")
+
+
+var logger := Log.get_logger("SteamRemovableMedia", Log.LEVEL.INFO)
+
+@onready var container: HFlowContainer = $%DriveCardContainer
+@onready var focus_group: FocusGroup = $%FocusGroup
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	logger.debug("I'm ready!")
+	steam_disks.drives_updated.connect(_on_drives_updated)
+	steam_disks.update_drives()
+
+
+func _on_drives_updated(devices: Array[BlockDevice]) -> void:
+	logger.debug("Got updated drives")#, steam_disks.device_tree)
+
+	# Clear the current grid of items
+	var keep_nodes := [focus_group]
+	for child in container.get_children():
+		if child in keep_nodes:
+			continue
+		container.remove_child(child)
+		child.queue_free()
+
+	# Poplulate drives
+	var last_focus: FocusGroup
+	for drive in devices:
```

Comment:
We should add a label with the text "No formattable disks available" and display it if the number of drivers is zero.